### PR TITLE
[FIX] PeakIntegrator: asymmetry factor and tailing factor formulas

### PR DESCRIPTION
**Pull request by pasqoo**
Incorrect formula used for the Tailing Factor and Peak Asymmetry peak shape metrics. Source for the formulas used previously was not correct.

### CODE REVIEW

**Comment by pasqoo** [comment]
About _msvc-15_ failure, some problems cloning the repo:
https://abibuilder.informatik.uni-tuebingen.de/jenkins/job/openms_pr_ci/1288/compiler=msvc-15,os_label=win10_64/console

**Comment by timosachsenberg** [comment]
restart jenkins

**Comment by pasqoo** [comment]
Same error

File: src/openms/source/ANALYSIS/OPENSWATH/PeakIntegrator.cpp [hunk 1]
```diff
@@ -316,9 +316,8 @@ namespace OpenMS
     psm.slope_of_baseline = (p.PosEnd(right) - 1)->getIntensity() - p.PosBegin(left)->getIntensity();
     psm.baseline_delta_2_height = psm.slope_of_baseline / peak_height;
     // other
-    psm.tailing_factor = psm.width_at_5 / std::min(peak_apex_pos - psm.start_position_at_5, psm.end_position_at_5 - peak_apex_pos);
```
[the remark this answers is not in the document]
Comment (dmccloskey) [reply src/openms/source/ANALYSIS/OPENSWATH/PeakIntegrator.cpp]: @pasqoo can you add these references in?
Asymmetry factor: 	http://www.lcresources.com/resources/TSWiz/hs170.htm
Tailing factor: http://www.lcresources.com/resources/TSWiz/hs160.htm

**Comment by pasqoo** [comment]
@timosachsenberg updated

**Comment by timosachsenberg** [comment]
+1 if passes

**Comment by pasqoo** [comment]
rebuild win

**Comment by timosachsenberg** [comment]
rebuild jenkins

**Comment by pasqoo** [comment]
There are two jobs in queue for this branch? https://abibuilder.informatik.uni-tuebingen.de/jenkins/job/openms_pr_ci/ Maybe one could be removed.

**Pull request merged**